warehouse/query-core/src/main/java/datawave/query/function/AncestorEquality.java

```diff
@@ -1,21 +1,29 @@
 package datawave.query.function;
 
-import datawave.data.hash.UIDConstants;
 import org.apache.accumulo.core.data.Key;
 
+import static datawave.data.hash.UIDConstants.DEFAULT_SEPARATOR;
+
 /**
- * A key equality implementation that compares to the root pointers of two doc Ids together.
- * 
- * For example, two IDs `h1.h2.h3.a.b.c.d` and `h1.h2.h3.e.f` would be considered equal by this check.
- * 
+ * A key equality implementation that compares key prefixes to determine ancestor equality.
  *
+ * For example, given a parent key "h1.h2.h3.a.b.c" a valid ancestor would be key "h1.h2.h3.a.b.c.d.e"
  */
 public class AncestorEquality implements Equality {
     
+    /**
+     * Determines if the first key an ancestor of the other key by checking for a common prefix.
```

Comment:
Determines if the first key **is** an ancestor of the other key by checking for a common prefix.

Reply:
Fixed